simpleflow/swf/executor.py

```diff
@@ -67,9 +67,10 @@ def run_fake_child_workflow_task(domain, task_list, workflow_type_name,
                                  workflow_type_version, workflow_id, input=None, result=None,
                                  child_policy=None, control=None, tag_list=None):
     conn = ConnectedSWFObject().connection
-    conn.start_child_workflow_execution(
-        workflow_type_name, workflow_type_version, workflow_id,
+    conn.start_workflow_execution(
```

Comment:
It exists in the `Layer1Decisions` objects, not in `Layer1`

Starting a child workflow: isn't it just this decision, when received by a decider, that instructs it to start an independent workflow (with `parentInitiatedEventId` and `parentWorkflowExecution`)?

(Could be totally confused)